pagopa/pn-frontend
# refactor/PN-13118 - Remove all references to SERCQ privacy policy for PF and PG

## Short description
The code contained in this PR takes care to remove every reference to SERCQ privacy policy for both PF and PG web-apps.

## How to test
Login as both citizen and PG administrator and verify the disclaimer showed inside the SERCQ activation modal window doesn't contain any reference to privacy policy.
